platform/linux-generic/pktio/dpdk.c

```diff
@@ -891,7 +899,6 @@ static inline int mbuf_to_pkt_zero(pktio_entry_t *pktio_entry,
 
 	pkt_dpdk = pkt_priv(pktio_entry);
 	nb_pkts = 0;
-	pool = pkt_dpdk->pool;
```

Comment:
Surprisingly removing this read causes significant performance hit (~6.5%) on my test system. Easiest way to"fix" this seems to be to move `uint32_t supported_ptypes = pkt_dpdk->supported_ptypes;` line from the for loop below to here.

Reply:
v8.